src/components/CreateArea.jsx

```diff
@@ -4,46 +4,37 @@ import AddIcon from '@mui/icons-material/Add';
 import Zoom from '@mui/material/Zoom';
 
 function CreateArea(props) {
-  const [inputText, setInputText] = useState({
-    title: '',
-    content: '',
-  });
+  const [titleInput, setTitleInput] = useState('');
```

Comment:
Great work on making states more meaningful! A little note on variable naming. The variable name itself should describe the information contained in this variable. In this case, it has the title text, but I may suggest it contains a reference to the Title Input element from the name.  I'd recommend using `title` and `setTitle` accordingly. The same goes for the next line.